### DISCUSSION TITLE
How to actually delete the message sent by bot, rather than adding a deleted label to it

### DISCUSSION THREAD

**Discussion by fengkx** [telegraf/telegraf#1940] [Q&A] [answered]
I am using `ctx.telegram.deleteMessage` to delete the message sent by bot. It seems recently telegram change the behavior of detele message. Instead of making the whole message disappear, Now An deleted label added to it.
![image](https://github.com/telegraf/telegraf/assets/16515468/8a0e5774-14cb-4f87-9041-9c6a60a7f68c)
Is there any way to delete the whole message and make it invisible.

**Comment by MKRhere** [comment, marked as answer]
Are you using a custom client? This is not the behaviour of official clients.

**Reply by fengkx** [reply]
![Screenshot_2024-01-30-10-27-58-247_org.telegram.messenger-edit.jpg](https://github.com/telegraf/telegraf/assets/16515468/6a1f0e6d-7bae-4ba3-b315-87037f843812)

No, I am using the official telegram for Android installed on Google Play.

**Reply by MKRhere** [reply]
Even though this app has the correct name and version, this is not the official app's behaviour.

Is this the app you have installed?
https://play.google.com/store/apps/details?id=org.telegram.messenger
When you open this link, it should say something to the effect of "Open App" instead of "Install".
